src/main/java/kr/allcll/checkll/api/AdminService.java

```diff
@@ -0,0 +1,26 @@
+package kr.allcll.checkll.api;
+
+import kr.allcll.checkll.datasource.PastGradeDataSource;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AdminService {
+
+    private final PastGradeDataSource pastGradeDataSource;
+
+    public AdminService(PastGradeDataSource pastGradeDataSource) {
+        this.pastGradeDataSource = pastGradeDataSource;
+    }
+
+    public void setUpInfoByString(String credentialInfo) {
+        pastGradeDataSource.setCredential(credentialInfo);
+    }
+
+    public void startInitialSession() {
+        pastGradeDataSource.setUp();
+    }
+
```

Comment:
R: initial이 아닐 수도 있어요! 

```suggestion
    public void startSession() {
        pastGradeDataSource.setUp();
    }

```

Reply:
Service의 메서드명을 추상화 인터페이스 메서드명이랑 비슷하게 요청 시작, 요청 종료의 메서드명을 각각 setUpSession, shutDownSession 으로 바꿨어요
이건 어떨까요

Reply:
좋아요!